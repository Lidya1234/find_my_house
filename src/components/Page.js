import '../style/main.scss';
import '../style/style.css';

const Page = () => (
  <div className="page">
    <h6>Scroll horizontally</h6>
  </div>
);
export default Page;
