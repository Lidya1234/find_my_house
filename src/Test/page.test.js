import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Page from '../components/Page';
import '@testing-library/jest-dom/extend-expect';

describe('The navigation bar should contain', () => {
  it('House logo', () => {
    const { getByText } = render(
      <MemoryRouter>
        <Page />
      </MemoryRouter>,
    );
    const link = getByText('Scroll horizontally');
    expect(link).toBeInTheDocument();
  });
  const component = renderer.create(
    <MemoryRouter>
      <Page />
    </MemoryRouter>,
  ).toJSON();

  test('Matchs snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
