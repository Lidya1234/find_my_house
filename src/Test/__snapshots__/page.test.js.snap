// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The navigation bar should contain Matchs snapshot 1`] = `
<div
  className="page"
>
  <h6>
    Scroll horizontally
  </h6>
</div>
`;
